test/new-e2e/tests/agent-metrics-logs/log-agent/k8s-logs/file_tailing_test.go

```diff
@@ -113,17 +119,23 @@ func (v *k8sSuite) TestLongLogLine() {
 	}
 
 	_, err = v.Env().KubernetesCluster.Client().BatchV1().Jobs("default").Create(context.TODO(), jobSpcec, metav1.CreateOptions{})
-	assert.NoError(v.T(), err, "Could not properly start job")
+	require.NoError(v.T(), err, "Could not properly start job")
```

Comment:
Notes for the reviewers: requires stops the test, if this fail we cannot move on